tensorflow/compiler/xla/mlir_hlo/tests/Dialect/mhlo/hlo-legalize-to-linalg.mlir

```diff
@@ -4962,3 +4962,23 @@ func.func @feature_group_count_convolution(%arg0: tensor<2x14x12x2xf64>, %arg1:
     : (tensor<2x14x12x2xf64>, tensor<7x7x1x2xf64>) -> tensor<2x6x8x2xf64>
   return %0 : tensor<2x6x8x2xf64>
 }
+
+// -----
+// The following test is identitical to the previous one, except that the
+// `mhlo.convolution` op lacks the (optional) `stride` and `reverse` attribute.
```

Comment:
While we're here, how about replacing the references to attribute names with `window_strides` and `window_reversal`? Thank you!